pkg/bpf/bpf.go

```diff
@@ -105,7 +105,7 @@ const (
 // CreateMap creates a Map of type mapType, with key size keySize, a value size of
 // valueSize and the maximum amount of entries of maxEntries.
 // mapType should be one of the bpf_map_type in "uapi/linux/bpf.h"
-func CreateMap(mapType int, keySize, valueSize, maxEntries, flags uint32) (int, error) {
+func CreateMap(mapType int, keySize, valueSize, maxEntries, flags, innerID uint32) (int, error) {
```

Comment:
Please add in the description what's the innerID

Reply:
Sure although I guess mind as well add all the fields then.